docs/IE_DG/inference_engine_intro.md

```diff
@@ -16,17 +16,20 @@ For complete API Reference, see the [API Reference](usergroup29.html) section.
 Inference Engine uses a plugin architecture. Inference Engine plugin is a software component that contains complete implementation for inference on a certain Intel&reg; hardware device: CPU, GPU, VPU, etc. Each plugin implements the unified API and provides additional hardware-specific APIs.
 
 Modules in the Inference Engine component
----------------------------------------
+-----------------------------------------
 
 ### Core Inference Engine Libraries ###
 
 Your application must link to the core Inference Engine libraries:
 * Linux* OS:
-    - `libinference_engine.so`, which depends on `libinference_engine_transformations.so` and `libngraph.so`
-    - `libinference_engine_legacy.so`, which depends on `libtbb.so`
+    - `libinference_engine.so`, which depends on `libinference_engine_transformations.so`, `libtbb.so`, `libtbbmalloc.so` and `libngraph.so`
+    - `libinference_engine_legacy.so`, which depends on `libinference_engine.so` and its dependencies
```

Comment:
BTW, we don't need to link with `libinference_engine_legacy.so` since 2021.2. So, we can remove it from IE core libraries section

Reply:
Ok. Removed.